redaxo/src/core/lib/util/log_file.php

```diff
@@ -208,6 +208,9 @@ class rex_log_entry
     public function __construct($timestamp, array $data)
     {
         $this->timestamp = $timestamp;
+        foreach ($data as $key => $value) {
```

Comment:
Sollte das in createFromString sein statt im construct()?

Reply:
Hmm Stimmt, ich glaube, da wäre es besser.

Reply:
Warum? $data wird ja direkt übergeben - und dann würde ein getData() ohne die Zeilenumbrüche nicht mit ausgeben - das createFromString() kommt da ja nirgendwo zum Zug, oder?

Reply:
... andererseits holt man sich den aktuellen Eintrag ja über rex_log_file:current() ... vielleicht doch besser da

Reply:
Die Methode ist das Gegenstück zu `__toString()`, und wird verwendet beim Einlesen der Einträge aus der Logdatei. Daher passt es da besser.